linux_os/guide/system/logging/ensure_rsyslog_log_file_configuration/rsyslog_encrypt_offload_actionsendstreamdrivermode/tests/default_no_pass.fail.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+bash -x setup.sh
+
+if [[ -f encrypt.conf ]]; then
+  sed -i i/\$ActionSendStreamDriverMod//g /etc/rsyslog.d/encrypt.conf
+fi
+  sed -i i/\$ActionSendStreamDriverMod//g /etc/rsyslog.conf
```

Comment:
I see a lot of files here with no line breaks at the end.. just make sure those get added